### PR TITLE
feat: add mIgnore tag

### PR DESCRIPTION
**Pull request by Zyie**
adds the `mIgnore` tag which will cause the manifest to ignore the asset and not add it.

### CODE REVIEW

File: src/manifest/pixiManifest.ts [hunk 1]
```diff
@@ -118,14 +122,17 @@ function collectAssets(
     }
 
     const bundleAssets = localBundle.assets;
-
     const finalAssets = asset.getFinalTransformedChildren();
 
-    if (asset.transformChildren.length > 0)
+    if (asset.transformChildren.length > 0 && !asset.inheritedMetaData[options.tags!.mIgnore!])
```
Comment (GoodBoyDigital) [inline src/manifest/pixiManifest.ts:127]: I guess this is kind of an early out, but wandering if we need this check? as we are filtering just below anyways?

File: src/manifest/pixiManifest.ts [hunk 2]
```diff
@@ -118,14 +122,17 @@ function collectAssets(
...
     {
+        const nonIgnored = finalAssets.filter((finalAsset) => !finalAsset.inheritedMetaData[options.tags!.mIgnore!]);
```
Comment (GoodBoyDigital) [inline src/manifest/pixiManifest.ts:129]: lets rename to something like `finalManifestAssets` or something along those lines!

**Review by GoodBoyDigital — COMMENTED** [review]
couple of suggested tweaks! looking good!

**Comment by Zyie** [comment]
will follow up in separate PR

https://github.com/play-co/pixi-external/issues/89

**Pull request merged**